contracts/interfaces/IAxelarServiceGovernance.sol

```diff
@@ -6,71 +6,71 @@ import { IInterchainGovernance } from './IInterchainGovernance.sol';
 
 /**
  * @title IAxelarServiceGovernance Interface
- * @dev This interface extends IInterchainGovernance and IMultisigBase for multisig proposal actions
+ * @dev This interface extends IInterchainGovernance for operator proposal actions
  */
 interface IAxelarServiceGovernance is IInterchainGovernance {
-    error InvalidMultisigAddress();
+    error InvalidOperator();
     error NotApproved();
     error NotAuthorized();
 
-    event MultisigApproved(
+    event OperatorApproved(
```

Comment:
```suggestion
    event OperatorProposalApproved(
```
existing name didn't make sense, update other events as well

Reply:
Got it, done